src/components/calcite-link/calcite-link.e2e.ts

```diff
@@ -2,12 +2,11 @@ import { E2EPage, newE2EPage } from "@stencil/core/testing";
 import { accessible } from "../../tests/commonTests";
 
 describe("calcite-link", () => {
-  it("is accessible", async () =>
-    Promise.all([
```

Comment:
You could have put an await here right? I do like it without the Promise.all use though. 👍

Reply:
Technically, yes, but you can return the promise directly and not await on anything.